services/order.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict], username: str, date: datetime = None
+) -> None:
+
+    order_user = get_user_model().objects.get(username=username)
+    new_order = Order.objects.create(user=order_user)
+
+    if date:
+        new_order.created_at = date
+        new_order.save()
+
+    for ticket_data in tickets:
+        Ticket.objects.create(
+            row=ticket_data["row"],
+            seat=ticket_data["seat"],
+            movie_session_id=ticket_data["movie_session"],
+            order=new_order
+        )
+
+
+def get_orders(username: str = None) -> QuerySet:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]